cookieConsent/OneTrust/browser.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable class-methods-use-this */
+class OneTrust {
+  constructor(sourceConfig) {
+    this.sourceConfig = sourceConfig;
+  }
+
+  isEnabled(destConfig) {
+    // If user does not load onetrust sdk before loading rudderstack sdk
+    // we will not be filtering any of the destinations.
+
+    if (!window.OneTrust && !window.OnetrustActiveGroups) {
+      return true;
+    }
+
+    /**
+     * Structure of onetrust consent group destination config.
+     * 
+     * "oneTrustConsentGroup": [
```

Comment:
The key name for the collection and individual entries is identical. We should avoid this.

Reply:
This logic is in the webapp for creating this datastructure